sdk/servicebus/azure-servicebus/azure/servicebus/aio/_servicebus_session_receiver_async.py

```diff
@@ -66,6 +62,14 @@ class ServiceBusSessionReceiver(ServiceBusReceiver, SessionReceiverMixin):
     :keyword dict http_proxy: HTTP proxy settings. This must be a dictionary with the following
      keys: `'proxy_hostname'` (str value) and `'proxy_port'` (int value).
      Additionally the following keys may also be present: `'username', 'password'`.
+    :keyword str user_agent: The user agent that should be appended to the built-in user agent string.
```

Comment:
From the discussion OOB, doesn't it prepend, not append?

Reply:
updated, copy the words from azure-core library